libraries/stumbler/src/main/java/org/mozilla/mozstumbler/svclocator/services/log/DebugLogger.java

```diff
@@ -14,8 +15,21 @@
  */
 public class DebugLogger implements ILogger {
 
+    private static final String LOG_TAG = LoggerUtil.makeLogTag(DebugLogger.class);
+
     static {
-        ShadowLog.stream = System.out;
+        try {
+            @SuppressWarnings("all")
```

Comment:
That's a comment on ugly reflection.  Our ProductionLogger no-ops all debug level logging.